util/app-config/src/cli.rs

```diff
@@ -526,3 +527,18 @@ fn is_hex(hex: &str) -> Result<(), String> {
         Err("Must 0x-prefixed hexadecimal string".to_string())
     }
 }
+
+fn is_h256(hex: &str) -> Result<(), String> {
+    if hex.as_bytes()[..2] == b"0x"[..] {
+        let mut hd = [0u8; 32];
+        if let Err(e) = faster_hex::hex_decode(&hex.as_bytes()[2..], &mut hd) {
```

Comment:
`faster_hex::hex_decode(&hex.as_bytes()[2..])`

Reply:
get it!....a little suggestion, we may need a line of  usage comment on hex_check().

Reply:
change to quake's suggestion....